packages/core/asyncComputed/index.test.ts

```diff
@@ -0,0 +1,82 @@
+import { ref, computed, nextTick } from 'vue-demi'
+import { renderHook } from '../../_tests'
+import { asyncComputed } from '.'
+
+describe('computed', () => {
+  it('is lazy', async() => {
+    const func = jest.fn(() => 'data')
+
+    renderHook(() => {
+      const data = computed(func)
+
+      expect(func).not.toBeCalled()
+
+      expect(data.value).toBe('data')
+
+      expect(func).toBeCalledTimes(1)
+    })
+  })
+})
+
+describe('asyncComputed', () => {
+  test('it is not lazy', async() => {
+    const func = jest.fn(() => Promise.resolve('data'))
+
+    const data = asyncComputed(func)
+
+    expect(func).toBeCalledTimes(1)
+
+    expect(data.value).toBeUndefined()
+
+    await nextTick()
+
+    expect(data.value).toBe('data')
+  })
+
+  test('re-computes when dependency changes', async() => {
+    const counter = ref(1)
+    const double = asyncComputed(() => {
+      const result = counter.value * 2
+      return Promise.resolve(result)
+    })
+
+    expect(double.value).toBeUndefined()
```

Comment:
Might need to wrapper it with `renderHook` I think

Reply:
It may have been a timing issue with Promise.resolve and nextTick, as it was working for me before. Looks like it is resolved by always waiting two ticks.
Should `renderHook` be used when possible in tests? I do not know exactly when I should include it.

Reply:
I think we should always use `renderHook` if possible. Unlike Vue 3 that the APIs can be used standalone, Vue 2 is a bit tricky to do that (which is what we are using for testing, likely to move to Vue 3 or isomorphic in the future). Can you try with that and see if it still needs to invoking nextTick twice?

Reply:
Looks like it is still needed, it was failing locally without it after adding render hooks